Readme.md

```diff
@@ -78,7 +78,7 @@ channels/
 Common channel setting are stored in `_channel.yml` file. You can configure
 following parameters:
 
-* `chat_ids` - list of chats or channels to send your message to
+* `chat_ids` - list of chats or channels to send your message to. In case of slack DM this field should be empty.
```

Comment:
Can I do it non required?
Remove:
`check_field(channel_config, 'chat_ids', channel_file)`

Because now `_channel.yaml`:
```
chat_ids:
bot: slack_dm
```